src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -54,6 +55,7 @@ interface Props {
 }
 
 const ActionsPageEvent = ({
+  index: eventIndex,
```

Comment:
why we just not name this prop eventIndex?

Reply:
Sure :+1: